examples/notebooks/beam-ml/bigtable_enrichment_transform.ipynb

```diff
@@ -753,70 +758,41 @@
       "execution_count": null,
       "metadata": {
-        "id": "St07XoibcQSb",
         "colab": {
           "base_uri": "https://localhost:8080/",
           "height": 1000
         },
+        "id": "St07XoibcQSb",
         "outputId": "34e0a603-fb77-455c-e40b-d15b672edeb2"
       },
       "outputs": [
         {
-          "output_type": "display_data",
           "data": {
-            "application/javascript": [
-              "\n",
-              "        if (typeof window.interactive_beam_jquery == 'undefined') {\n",
-              "          var jqueryScript = document.createElement('script');\n",
-              "          jqueryScript.src = 'https://code.jquery.com/jquery-3.4.1.slim.min.js';\n",
-              "          jqueryScript.type = 'text/javascript';\n",
-              "          jqueryScript.onload = function() {\n",
-              "            var datatableScript = document.createElement('script');\n",
-              "            datatableScript.src = 'https://cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js';\n",
-              "            datatableScript.type = 'text/javascript';\n",
-              "            datatableScript.onload = function() {\n",
-              "              window.interactive_beam_jquery = jQuery.noConflict(true);\n",
-              "              window.interactive_beam_jquery(document).ready(function($){\n",
-              "                \n",
-              "              });\n",
-              "            }\n",
-              "            document.head.appendChild(datatableScript);\n",
-              "          };\n",
-              "          document.head.appendChild(jqueryScript);\n",
-              "        } else {\n",
-              "          window.interactive_beam_jquery(document).ready(function($){\n",
-              "            \n",
-              "          });\n",
-              "        }"
-            ]
+            "application/javascript": "\n        if (typeof window.interactive_beam_jquery == 'undefined') {\n          var jqueryScript = document.createElement('script');\n          jqueryScript.src = 'https://code.jquery.com/jquery-3.4.1.slim.min.js';\n          jqueryScript.type = 'text/javascript';\n          jqueryScript.onload = function() {\n            var datatableScript = document.createElement('script');\n            datatableScript.src = 'https://cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js';\n            datatableScript.type = 'text/javascript';\n            datatableScript.onload = function() {\n              window.interactive_beam_jquery = jQuery.noConflict(true);\n              window.interactive_beam_jquery(document).ready(function($){\n                \n              });\n            }\n            document.head.appendChild(datatableScript);\n          };\n          document.head.appendChild(jqueryScript);\n        } else {\n          window.interactive_beam_jquery(document).ready(function($){\n            \n          });\n        }"
```

Comment:
would it make sense to clear the outputs in the notebook?

Reply:
yeah, I may have missed this last time